Test/SqlDom/TSqlParserTest.cs

```diff
@@ -505,6 +505,24 @@ public void OpenRowsetBulkWithOneFile()
             ));
         }
 
+        [TestMethod]
+        [Priority(0)]
+        [SqlStudioTestCategory(Category.UnitTest)]
+        public void Object_IDParsingIn160ParserTest()
```

Comment:
You can add a test to [Only160SyntaxTests](https://github.com/microsoft/SqlScriptDOM/blob/main/Test/SqlDom/Only160SyntaxTests.cs) instead of writing a new one. It runs parser and script generator. You just have to add a sql file to TestScripts and Baselines160 folder.

Reply:
Done